extensions/spotify/init.lua

```diff
@@ -27,10 +42,23 @@ end
 ---
 --- Returns:
 ---  * None
-function spotify.play()
+function spotify.playpause()
   tell('playpause')
 end
 
+--- hs.spotify.play()
+--- Function
+--- Play current spotify track
```

Comment:
Small detail, but the single line description should be more in the form of "Plays the current Spotify track"